packages/frontend/app/components/curriculum-inventory/new-sequence-block.js

```diff
@@ -114,6 +143,82 @@ export default class CurriculumInventoryNewSequenceBlock extends Component {
     });
   }
 
+  @cached
+  get academicLevelsData() {
+    return new TrackedAsyncData(this.args.report.academicLevels);
+  }
+
+  get academicLevels() {
+    return this.academicLevelsData.isResolved ? this.academicLevelsData.value : [];
+  }
+
+  @cached
+  get defaultStartingAcademicLevelData() {
+    return new TrackedAsyncData(
+      this.getDefaultStartingAcademicLevel(this.args.report, this.args.parent),
+    );
+  }
+
+  get defaultStartingAcademicLevel() {
+    return this.defaultStartingAcademicLevelData.isResolved
+      ? this.defaultStartingAcademicLevelData.value
+      : null;
+  }
+
+  async getDefaultStartingAcademicLevel(report, parent) {
```

Comment:
Not sure if this warrants its own method, could just be a `() =>` closure instead.

Reply:
i couldn't get this to work with an async fat arrow function that's instantly called inside the TrackedAsyncData constructor.